.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,38 @@
+# This GitHub Actions workflow contains one job:
+# -- Build MKDocs uses the latest Ubuntu image and Python 3.9 to build
+# the MKDocs website using the latest Poetry version.
+
+name: Build MKDocs
+
+on:
+  push:
+    branches:
+      -main
```

Comment:
I don't think this PR should fix this, but I think you'll probably want two separate jobs for MkDocs -- one for actually building and publishing documentation on merges to `main`, and another for linting the configuration and files (attempting a build, essentially) that runs on pull requests.

Reply:
Noted for future work.